src/components/UI/Input/Input.js

```diff
@@ -6,14 +6,38 @@ const input = props => {
 
   switch (props.elementType) {
     case 'input':
-      inputElement = (
-        <input
-          className={inputClasses.join(' ')}
-          {...props.elementConfig}
-          value={props.value}
-          onChange={props.changed}
-        />
-      );
+      if ('radio' === props.elementConfig.type) {
```

Comment:
general pattern of special/specific code in generalized component is a bad Idea:
https://github.com/crisanlucid/react-quiz/blob/c3ca710db370d6c57130a2a964b72ac2f0dc460d/src/components/UI/Input/Input.js#L9-L39